src/test/resources/features/AppCenter/AppCenterUser.feature

```diff
@@ -163,8 +154,7 @@ Feature: To be able to access applications
     And I go to Administer application center Page
     And I enable Mandatory 'Tasks' for application
 
-  @appcenter
-  @appcenterUser
+  @failing
```

Comment:
please remove this tag

Reply:
This new tag will be used to disable failing tests willing to be updated/fixed.
In fact, inside the CI/CD jobs we will use the command `not @ignored and not @failing` to execute test cases.
We will not use tag `@ignored` on failing tests to unblock the CI/CD anymore. It will be meant and used only on Tests that we will ignore for a while.

Reply:
ok well received , thanks